doc/WEBUI.md

```diff
@@ -202,6 +202,7 @@ single or double quotes.
   - `netdev`, `networkdevice` look for network devices (firewalls,
     routers, ...).
   - `phonedev` look for telephony devices.
+  - `cpe:[value] | cpe:[field=value](, ...)` look for a given cpe, by value or by field (type, vendor, component or product).
```

Comment:
Don't you think a syntax like `cpe:field:value,field:value` would better fit the syntax used by other filters?